### PR TITLE
Device manager - expandable session details in device list (PSG-644)

### PR DESCRIPTION
**Pull request by kerryarchibald**
Includes #9181

https://user-images.githubusercontent.com/3055605/184679806-e515821d-7e19-49aa-b3d8-c8eb707304aa.mov

## Checklist

* [ ] Tests written for new code (and old code if feasible)
* [ ] Linter and other CI checks pass
* [ ] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

Notes: Device manager - expandable session details in device list

---
Here's what your changelog entry will look like:

## ✨ Features
 * Device manager - expandable session details in device list ([\#9188](https://github.com/matrix-org/matrix-react-sdk/pull/9188)). Contributed by @kerryarchibald.

### CODE REVIEW

**Review by weeman1337 — COMMENTED** [review]
It is a bit fiddly to hit the arrow on the right. Can the entire device item be clickable?

Otherwise :+1: I like the new session management

**Comment by kerryarchibald** [comment]
@weeman1337 hmm, they will get a checkbox on the left soon too, not sure which should be the dominant action. I'll ask Jano about that.
Screenshot 2022-08-16 at 16 26 49

File: src/components/views/settings/devices/DeviceDetails.tsx [hunk 1]
```diff
@@ -49,9 +49,10 @@ const DeviceDetails: React.FC<Props> = ({ device }) => {
             ],
         },
     ];
-    return <div className='mx_DeviceDetails'>
+    return <div className='mx_DeviceDetails' data-testid={`device-detail-${device.device_id}`}>
         <section className='mx_DeviceDetails_section'>
             <Heading size='h3'>{ device.display_name ?? device.device_id }</Heading>
+            <br />
```
[the remark this answers is not in the document]
Comment (kerryarchibald) [reply src/components/views/settings/devices/DeviceDetails.tsx]: Lazy way to vertically space these two, replaced with some css.

**Comment by weeman1337** [comment]
> @weeman1337 hmm, they will get a checkbox on the left soon too, not sure which should be the dominant action. I'll ask Jano about that.

Ok. Let's keep going. We can change this later.

**Review by weeman1337 — APPROVED** [review]

**Pull request merged**